src/meta/src/rpc/metrics.rs

```diff
@@ -394,6 +398,18 @@ impl MetaMetrics {
         )
         .unwrap();
 
+        let total_object_count = register_int_gauge_with_registry!(
+            "storage_total_object_count",
+            "Total number of objects that includes dangling objects. Note that the metric is updated right before full GC. So subsequent full GC may reduce the actual value significantly, without updating the metric.",
+            registry
+        ).unwrap();
+
+        let total_object_size = register_int_gauge_with_registry!(
+            "storage_total_object_size",
+            "Total size of objects that includes dangling objects. Note that the metric is updated right before full GC. So subsequent full GC may reduce the actual value significantly, without updating the metric.",
```

Comment:
Shall we need to trigger `FullScanTask` only for updating metrics periodically? (More frequent updates)